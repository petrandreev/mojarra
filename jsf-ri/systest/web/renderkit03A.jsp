<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->

<HTML>

    <%@ taglib uri="http://java.sun.com/jsf/html" prefix="h" %>
    <%@ taglib uri="http://java.sun.com/jsf/core" prefix="f" %>

       <f:view>  
          <h:outputText id="simplelabel" value="This IS TEXT FROM THE CUSTOM RENDERER" /> 
       </f:view>

</HTML>
