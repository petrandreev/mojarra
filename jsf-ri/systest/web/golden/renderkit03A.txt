

<HTML>

    
    

       <span id="simplelabel">This IS A SIMPLE LABEL FROM THE CUSTOM RENDERER</span>

</HTML>


/renderkit03.jsp PASSED

